tests/cases/eth2eth/scripts/test-tx

```diff
@@ -5,37 +5,75 @@ set -eu
 SCRIPT_DIR=$(cd $(dirname ${BASH_SOURCE:-$0}); pwd)
 source ${SCRIPT_DIR}/util/relayer-util
 
-BALANCE_GO=${SCRIPT_DIR}/../../../chains/ethereum/scripts/cmd/balance/main.go
-TRANSFER_GO=${SCRIPT_DIR}/../../../chains/ethereum/scripts/cmd/transfer/main.go
+ICS20_CLI="ethereum-ics20-cli"
 
-SIMPLE_TOKEN_ADDRESS="0xff77D90D6aA12db33d3Ba50A34fB25401f6e4c4F"
+TMP_DIR=/tmp
+if ! which ${ICS20_CLI} > /dev/null 2>&1; then
+  if [ ! -d "${TMP_DIR}" ]; then
+    mkdir -p ${TMP_DIR}
+  fi
+  git clone https://github.com/datachainlab/ethereum-ics20-cli.git ${TMP_DIR}/ethereum-ics20-cli
+  cd ${TMP_DIR}/ethereum-ics20-cli
+  go install
+  export PATH=$PATH:$(go env GOPATH)/bin
+fi
+
+SIMPLE_TOKEN_ADDRESS="0xff77d90d6aa12db33d3ba50a34fb25401f6e4c4f"
 ICS20_TRANSFER_BANK_ADDRESS="0xaE1C9125BbcF63bf51294C4D15CBD472782E330D"
 ICS20_BANK_ADDRESS="0x2F5703804E29F4252FA9405B8D357220d11b3bd9"
 
+CHAIN_A_JSON=${CONF_DIR}/chains/ibc-0.json
+CHAIN_B_JSON=${CONF_DIR}/chains/ibc-1.json
+PATH_JSON=${CONF_DIR}/path.json
+
+CHAIN_A_ID=$(jq -r '.chain."eth_chain_id"' ${CHAIN_A_JSON})
+CHAIN_B_ID=$(jq -r '.chain."eth_chain_id"' ${CHAIN_B_JSON})
+
+MNEMONIC_A=$(jq -r '.chain."hdw_mnemonic"' ${CHAIN_A_JSON})
+MNEMONIC_B=$(jq -r '.chain."hdw_mnemonic"' ${CHAIN_B_JSON})
+
+RPC_ADDRESS_A=$(jq -r '.chain."rpc_addr"' ${CHAIN_A_JSON})
+RPC_ADDRESS_B=$(jq -r '.chain."rpc_addr"' ${CHAIN_B_JSON})
+
+PORT_ID=$(jq -r '.dst."port-id"' ${PATH_JSON})
+CHANNEL_ID=$(jq -r '.dst."channel-id"' ${PATH_JSON})
+
+WALLET_ADDRESS_A=$(${ICS20_CLI} wallet address --mnemonic="${MNEMONIC_A}" --wallet-index=1)
+WALLET_ADDRESS_B=$(${ICS20_CLI} wallet address --mnemonic="${MNEMONIC_B}" --wallet-index=2)
+
 show_balance() {
   state=$1
-  user_id=$2
-  user_name=$3
-  result=$(go run ${BALANCE_GO} ${CONF_DIR} ${user_id} ${SIMPLE_TOKEN_ADDRESS} ${ICS20_TRANSFER_BANK_ADDRESS} ${ICS20_BANK_ADDRESS})
-  result1=$(echo $result | cut -d ',' -f 1)
-  result2=$(echo $result | cut -d ',' -f 2)
-  echo "${state} ibc0 balance of ${user_name}: $result1"
-  echo "${state} ibc1 balance of ${user_name}: $result2"
+  chain=$2
+  user_wallet=$3
+  token_address=$4
+  user_name=$5
+  balance=$(${ICS20_CLI} balance --rpc-address=${chain} --ics20-bank-address=${ICS20_BANK_ADDRESS} --wallet-address=${user_wallet} --token-address=${token_address})
+  echo "${state} ibc0 balance of ${user_name}: $balance"
```

Comment:
`chain` is not necessarily `ibc0`.

Reply:
https://github.com/datachainlab/yui-relayer-build/pull/9/commits/4cb346eabed7ceb13a2b9192085503d238967bf7
change to ${chain}  (rpc_address like http://localhost:8645)